docs/section19/19-19.md

```diff
@@ -0,0 +1,111 @@
+                - Else if the system type is variable volume and not system 11: `Elif any(baseline_system_type_compare(sys_type, target_sys_type, false) for target_system_type in [HVAC_SYS.SYS_5, HVAC_SYS.SYS_6, HVAC_SYS.SYS_7, HVAC_SYS.SYS_8]): `  
+                    - Calculate the expected BHP (if no other pressure drop adjustments apply): `expected_BHP = (0.0013 * supply_cfm_b) + A`  
+                    - Calculate the minimum BHP: `minimum_BHP = (0.0013 * supply_cfm_b)`
+                - Else, the system is system 11: `Else:`  
+                    - Calculate the expected BHP (if no other pressure drop adjustments apply): `expected_BHP = (0.00062 * supply_cfm_b) + A`  
+                    - Calculate the minimum BHP: `minimum_BHP = (0.00062 * supply_cfm_b)`
+
+                - Get the motor efficiency from Table G3.9.1 for expected BHP (should be looked up based on the next motor size greater than the bhp (i.e., expected_BHP)): `motor_eff_ex = data_lookup(G3.9.1,expected_BHP)`  
+                - Get the motor efficiency from Table G3.9.1 for minimum BHP (should be looked up based on the next motor size greater than the bhp (i.e., expected_BHP)): `motor_eff_min = data_lookup(G3.9.1,minimim_BHP)`  
+                - Calculate the expected Wattage for expected BHP (assuming no other pressure drop adjustments apply except energy recovery and MERV filters): `fan_Wattage_expected = expected_BHP * 746 *(1/motor_eff_ex)`  
+                - Calculate the minimim Wattage for minimim BHP: `fan_Wattage_minimim = minimim_BHP * 746 *(1/motor_eff_min)`  
+
+
+            - **Rule Assertion:** 
+            - Case 1: If fan wattage expected equals total fan power: `if more_than_one_exhaust_fan_and_energy_rec_is_relevant == false and more_than_one_supply_fan == false and total_fan_power == fan_Wattage_expected: outcome = "PASS"`  
+            - Case 2: Elif conservative comparison equals true and fan is less than expected outcome is Pass:`elif more_than_one_supply_fan == false and AHJ_RA_compare == True and total_fan_power < fan_Wattage_expected: PASS and raise_message "The total fan power for <insert hvac.id> is modeled as <insert total_fan_power> kW which is less than the expected including pressure drop adjustments for exhaust air energy recovery and MERV filters as applicable which was calculated as <insert fan_Wattage_minimim> kW. Pass because this is generally considered more conservative."`  `  
+            - Case 3: Elif conservative comparison is false and fan is less than expected outcome is Fail:`elif more_than_one_supply_fan == false and AHJ_RA_compare == False and total_fan_power < fan_Wattage_expected: "FAIL" and raise_message "The total fan power for <insert hvac.id> is modeled as <insert total_fan_power> kW which is less than the expected including pressure drop adjustments for exhaust air energy recovery and MERV filters as applicable which was calculated as <insert fan_Wattage_minimim> kW ."`         
```

Comment:
Can't it just be that coservative comparison is false and fan is not equal to expected outcome then FAIL?

Reply:
@mkarpman we do not know if there are other relevant pressure drop adjustments so thought it might not be wise to have such a stringent fail outcome.

Reply:
OK, I see. So if it's more than the outcome would be UNDERTERMINED.